tests/testArgumentParser.py

```diff
@@ -289,13 +290,15 @@ def testAtFile(self):
     def testLogDest(self):
         """Test --logdest
         """
-        logFile = "tests_argumentParser_testLog_temp.txt"
-        self.ap.parse_args(
-            config = self.config,
-            args = [DataPath, "--logdest", logFile],
-        )
-        self.assertTrue(os.path.isfile(logFile))
-        os.remove(logFile)
```

Comment:
Thanks for finding the issue. This is removing the log file but not closing the file, hence the `.nfs` file still sitting around. I'm not entirely sure why this is not seen as a leak in the non-NFS runs. It might be that this is a file descriptor leak but `psutil` does not see it because it is not associated with a file name any more. On NFS it is an actual file so turns up as a leak.